tests/models/test_modeling_mamba2.py

```
@@ -0,0 +1,35 @@
+
+# ===================================================================================
+# Test for Modeling (Forward/Backward Pass)
+# ===================================================================================
+@pytest.mark.parametrize("L", [4])
+@pytest.mark.parametrize("B", [4])
+@pytest.mark.parametrize("T", [1024])
+@pytest.mark.parametrize("H", [4])
+@pytest.mark.parametrize("D", [64, 128])
+@pytest.mark.parametrize("dtype", [torch.bfloat16])
+@pytest.mark.parametrize("use_l2warp", [True, False])
+def test_mamba2_modeling(L, B, T, H, D, dtype, use_l2warp):
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)